torax/sources/tests/source.py

```diff
@@ -252,10 +252,38 @@ def test_defaults_output_zeros(self):
           profile,
           source_lib.ProfileType.CELL.get_zero_profile(geo),
       )
+    with self.subTest("prescribed"):
```

Comment:
nit - use single-quotes because that's whats in the rest of the file, for consistency.

Reply:
Resolved in 8cd2393379ebaed4ed94ac327d1bbffd65c3c4d1